config.yml

```diff
@@ -13,4 +13,14 @@ nerf-endermen-drops: false
 nerf-pigzombie-drops: false
 animal-chunk-cap: 30
 endpoint: http://standardsurvival.com/api/v1/
-rts-address: http://rts.standardsurvival.com
\ No newline at end of file
+rts-address: http://rts.standardsurvival.com
+
+muted-words:
+- 'nigger'
```

Comment:
leave this list empty, I don't want these words committed in the repo

Reply:
Sorry about that, should've thought about that